flake8_future_import.py

```diff
@@ -150,7 +178,7 @@ def main(args):
         ignored = set()
     for filename in args.files:
         with open(filename, 'rb') as f:
-            tree = compile(f.read(), filename, 'exec', PyCF_ONLY_AST)
+            tree = ast.parse(f.read(), filename=filename, mode='exec')
```

Comment:
If you are familiar with git's rebase I'd appreciate if that is separated into a separate pull request. I need to look at it more closely as it should then never warn that `print_function` is missing and I didn't really notice that issue (so either I'm blind or something else is going on).

Now I'm mentioning `git rebase` because I'd appreciate if here is no commit actually changing that line but that the commits of this PR are based on the new PR. Alternatively you could split this in two commits but have here the same PR. That would just mean that this change won't be merged sooner.